main/create-window.js

```diff
@@ -86,9 +86,13 @@ module.exports = function createWindow (options) {
   }
 
   function updateTitle () {
-    var prefix = fromFile ? (path.basename(options.filePath) + ' - ') : ''
+    if (options.title) {
+      win.setTitle(options.title)
```

Comment:
It was intentional, however this can be discused.
Adding ` - vmd` isn't necessary for my purpose. `vmd` is mentioned in menu btw so user do not forget these 3 letters :-)